packages/salesforcedx-apex-replay-debugger/src/core/heapDumpService.ts

```diff
@@ -134,7 +134,8 @@ export class HeapDumpService {
                 const updatedVariable = this.createVariableFromReference(
                   symbolName,
                   refVar,
-                  new Map<string, null>()
+                  new Map<string, null>(),
```

Comment:
The `replaceFrameVariablesWithHeapDump` is _quite_ long. 
It is worth considering breaking this up into smaller methods, which each method having a more descriptive name than having all the comments.

I don't know if you have time to do this now but, in general, this code type of long method code is not maintainable.

Reply:
I can create a work item to do this but it's not happening right now. There are other bugs that need to get immediately looked at from the Blitz.